spec/integration/riak/bucket_types_spec.rb

```diff
@@ -92,19 +92,18 @@
       describe 'deletion' do
         it 'self-deletes with a bucket type' do
           expect(untyped_object).to be # ensure existence
-
-          expect(object.delete).to be
-          expect{ object.reload }.to raise_error /not_found/
-          expect(untyped_object).to be
           expect{ untyped_object.reload }.to_not raise_error
+
+          expect(untyped_object.delete).to be
+          expect{ untyped_object.reload }.to raise_error /not_found/
         end
 
         it 'deletes from the typed bucket' do
           expect(untyped_object).to be # ensure existence
-
-          expect(bucket.delete object.key).to be
-          expect{ object.reload }.to raise_error /not_found/
```

Comment:
The goal here is to use the typed `bucket` to delete `object` without touching `untyped_object`.